### ISSUE TITLE
OS_Info.full_name on MacOS

### ISSUE THREAD

**Issue by bart9h** [ThePhD/infoware#53]
OS_Info.full_name on MacOS gives me information about the kernel (basically `uname -a`).

On Linux it gives information (name and version) about the distribution, and that looks more coherent.

Shouldn't the MacOS version do something similar, like the output of the `sw_vers` command?

**Comment by nabijaczleweli** [comment]
Good point; according to [this old `sw_vers`](https://opensource.apple.com/source/DarwinTools/DarwinTools-1/sw_vers.c.auto.html) and snooping on my VM, this should be extractable from the `/System/Library/CoreServices/SystemVersion.plist` (and/or `/ServerVersion.plist` beforehand, if it exists maybe? who knows) list. Sounds like an easy port for someone who's used CoreFoundation dictionaries before.
